pkg/crc/machine/machine.go

```diff
@@ -284,20 +284,6 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 		return startError(startConfig.Name, "Error getting the IP", err)
 	}
 
-	var hostIP string
```

Comment:
It was added when we tried to use the coredns and never removed this dead code :+1:

Reply:
can this be added to the commit log?

Reply:
done :+1: